Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/XldbDataStore.cs

```diff
@@ -18,6 +19,7 @@ public sealed class XldbDataStore : IDisposable
         /// </summary>
         private KeyValueStoreAccessor Accessor { get; set; }
         private Dictionary<ExecutionEventId, MessageParser> m_eventParserDictionary = new Dictionary<ExecutionEventId, MessageParser>();
+        public static string EventCountKey = "EventCount";

```

Comment:
>static [](start = 15, length = 6)

const instead of a static? you are not writing into that field, right?

Reply:
yea was not going to write into it. Would const or static readonly be better?

---
In reply to: [310239344](https://github.com/microsoft/BuildXL/pull/694#discussion_r310239344) [](ancestors = 310239344)

Reply:
I'd use const

---
In reply to: [310243113](https://github.com/microsoft/BuildXL/pull/694#discussion_r310243113) [](ancestors = 310243113,310239344)

Reply:
yes, either const or `readonly` static